test/curve/poolData.json

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
Is it limited to include only the top most used pools?

Reply:
it is now. top 25